nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenTCP.java

```diff
@@ -291,4 +305,15 @@ protected String getBatchKey(ByteArrayMessage event) {
         }
         return eventBatcher;
     }
+
+    private void addClientCertificateAttributes(final Map<String, String> attributes, final ByteArrayMessage event) {
+        final SslSessionStatus sslSessionStatus = event.getSslSessionStatus();
+        if (sslSessionStatus != null) {
+            attributes.put(CLIENT_CERTIFICATE_SUBJECT_DN_ATTRIBUTE, sslSessionStatus.getSubjectDN());
+            attributes.put(CLIENT_CERTIFICATE_ISSUER_DN_ATTRIBUTE, sslSessionStatus.getIssuerDN());
+        } else {
+            getLogger().debug("Remote Peer [{}] not verified: client certificates not provided",
+                    event.getSender());
+        }
```

Comment:
Having the debug log seems unnecessary since the absence of the attributes in outgoing FlowFiles indicates the same fact of missing client certificate information.
```suggestion
        }
```